uber/utils.py

```diff
@@ -198,7 +202,7 @@ def to_dict(self):
 
     @property
     def models(self):
-        return [self.from_sessionized(d) for d in self.targets]
+        return self._models_cached
```

Comment:
An easier approach would have been to just change this from a ``@property`` to a ``@cached_property`` which would have implemented this pattern automatically under the hood.